packages/data-service-generator/project.json

```diff
@@ -80,6 +80,14 @@
         "envFile": "packages/data-service-generator/.env"
       }
     },
+    "generate-local-code-only": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "ts-node -P tsconfig.app.json -r tsconfig-paths/register ./src/main.ts ",
+        "cwd": "packages/data-service-generator",
+        "envFile": "packages/data-service-generator/.env"
+      }
+    },
```

Comment:
this might become irrelevant with this change merged:
https://github.com/amplication/amplication/pull/5193/files#diff-1516d412c4cb5830f51541422e3b6fc06c855126c307ec0cc2d763baac4702aaL90

Reply:
This is not a duplicate of `generate-local-code`. Please remove the duplicated target